styleguide/source/_patterns/03-organisms/poll.twig

```diff
@@ -0,0 +1,26 @@
+<section class="joe__poll {% if poll.inline %}joe__poll--inline{% endif %}">
```

Comment:
Yes I made a separate ticket for that since I want to see what the drupal markup is and style that for ease of implementation. I can't recall the ticket number off the top of my head but I think it is in two sprints maybe.